Xamarin.Essentials/MediaPicker/MediaPicker.shared.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace Xamarin.Essentials
+{
+    public static partial class MediaPicker
+    {
+        public static bool IsPhotoCaptureAvailable
+            => PlatformIsPhotoCaptureAvailable;
```

Comment:
Is this just photo? Or photo and video? Maybe should be `IsMediaCaptureAvailable`?

Reply:
Switched this to `IsCaptureAvailable`